jscomp/super_errors/super_misc.ml

```diff
@@ -73,82 +69,96 @@ ppf
   | Some n -> n
   in
   (* coloring *)
-  let (highlighted_line_number, highlighted_open_tag): (string -> string -> unit, Format.formatter, unit) format * (unit, Format.formatter, unit) format = 
-    if is_warning then ("@{<info>%s@}@{<dim> @<1>%s @}", "@{<info>")
-    else ("@{<error>%s@}@{<dim> @<1>%s @}", "@{<error>")
+  let (highlighted_line_number, highlighted_open_tag): 
+    (string -> (Format.formatter -> unit -> unit) -> unit -> unit, Format.formatter, unit) format * (unit, Format.formatter, unit) format 
+    = 
```

Comment:
annotated with `_ format` is good enough? with such a complete type, it is  hard to do refactoring later